src/control/SConscript

```diff
@@ -118,9 +118,10 @@ def install_go_bin(denv, gosrc, libs, name, install_name):
     if 'GOCACHE' in os.environ:
         denv['ENV']['GOCACHE'] = os.environ['GOCACHE']
     target = daos_build.run_command(denv, build_bin, src, libs,
-                                    f'cd {gosrc}; {GO_BIN} build -v -mod vendor {go_ldflags()} ' +
-                                    f'{get_build_flags(denv)} {get_build_tags(denv)} ' +
-                                    f'-o {build_bin} {install_src}')
+                                    f'cd {gosrc}; {GO_BIN} build -v -mod vendor {go_ldflags()} '
+                                    + f'{get_build_flags(denv)} '  # noqa: W503
```

Comment:
This only happens with the githook and I could not figure out how to silence it other than to ignore it.

Reply:
This is a weird one and it's to do with where the + goes, on the end of the previous line or the beginning of this one.

Flake checks for and warns on both, (W503 and W504), and what is considered best practice has changed over time so the expectation seems to be you pick what is correct for your project and disable the "wrong" check, depending on if you're writing new or editing legacy code.
For the codebase as a whole our code is close to 50/50 but with many hundred instances of each, for the scons files however I just checked and there are 0 of one type and 5-6 of the other.

You should add something to https://github.com/daos-stack/daos/blob/master/.flake8-scons#L16 and not use the noqa here.